Unfettered Hai side-story: Acorn Cookies. (#10808)
endless-sky/endless-sky: data/hai/hai missions.txt

			`Hareek was quite tight-lipped about what he planned to do once on <planet>. The most you could get out of him was more stories about his past as a military pilot. Once you land, Hareek jumps off your ship, moving surprisingly fast for his age.`
			`"Thank you, Captain, for making this journey. I apologize for not telling you more about me; the work I do is not something that I can reveal to any human who asks." He hands you <payment> and moves away. The surrounding Unfettered appear almost awed by his presence and consequentially give you a wide berth.`
		log "Minor People" "Hareek" `An old Hai who claims to foresee the future. Dropped him off in Unfettered space so he could spread his message there. He appears to be well-respected amongst the Unfettered.`

mission "Acorn Delights: Part 1"
	minor
	name "Acorns for Anoo"
	description "Bring Eeeya to <destination> to deliver <cargo> to her missing granddaughter."
	source "Allhome"
	destination "Stonebreak"
	passengers 1
	cargo "acorn cookies" 3
	to offer

> 		random < 85

Author: ```suggestion
		random < 35
```

	source "Darkcloak"
	destination "Allhome"
	passengers 1
	to offer
		has "Acorn Delights: Part 3: done"
	on offer
		payment 3000
		log "Finally found Anoo. It turns out she joined the Unfettered. I am supposed to take Eeeya to <destination> now."
		conversation
			`You're ushered from the landing platform to a nearby hospital, where Eeeya finally manages to convey your intent. A few Unfettered Hai nearby are muttering in their language, and you keep hearing the Hai word for "cookies" as they watch Eeeya. Along the way, you notice an odd contrast of squalor and advanced machinery. Unlike their southern brethren, it seems the Unfettered value function and progress over aesthetics and comfort.`
			`	As you rise through the main tower of the hospital, you're surprised at how advanced it is. Despite their willingness to die in battle, the Unfettered seem to value the lives of the survivors. Perhaps this hospital is to heal them quickly so that they can fight again as soon as possible.`
			`	You reach a high floor filled with personal offices. Eeeya knocks on each door, apologizing when the Unfettered inside aren't who she's looking for, until you finally reach the very last room.`

> 			`	Eeeya opens this door, and runs to the Hai inside it - Anoo. A few minutes later, they begin to speak, surprisingly in a language you understand. Eventually the conversation turns to Anoo's new occupation.`

Reviewer: `A few minutes later, they begin to speak, surprisingly in a language you understand.`
This is pretty surprising. Why would they have this personal conversation in the human language?
Maybe add something here about them having this conversation in humanese for the benefit of the player?
```suggestion
			`	Eeeya opens this door, and runs to the Hai inside it - Anoo. A few minutes later, they begin to speak in Hai, then remember that you are in the room.`
			`"Please, Anoo, let's speak in the human language. <first> came all this way for me, and it's only fair that the Captain be able to understand what's going on."`
			 `The conversation then turns to Anoo's new occupation.`
```

Author: ```suggestion
			`	Eeeya opens this door, and runs to the Hai inside it - Anoo. A few minutes later, they begin to speak in Hai, then remember that you are in the room.`
			`	"Please, Anoo, let's speak in the human language. <first> came all this way for me, and it's only fair that the Captain be able to understand what's going on."`
			 `	The conversation then turns to Anoo's new occupation.`
```

			`	"We think of them as mindless brutes, but they're not! They have scientists, doctors, musicians, and writers. Our resources dwarf theirs, but they thrive with what little they have. Soon the Unfettered will have many more planets and almost unlimited resources. The era of scarcity will be over, and they'll have the power to choose their future.`
			`	"If enough good people join the Unfettered, we can change their minds and set them on a more productive path. They'll need good doctors."`
			`	"But-" Eeeya objects.`
			choice
				`	"Anoo is right; the Unfettered are a noble people. I would recommend joining them."`
					goto joinuhai
				`	"There's no good reason to join the Unfettered. What have they ever done except kill innocent Hai?"`
					goto "don't join"
			label joinuhai
			`	"Well... I really don't know..."`
			choice
				`	"Never mind, you're right. Joining the Unfettered is a bad idea after all."`
				`	"I stand by their guns. The Unfettered are honorable."`
					goto reallyjoin
			label "don't join"

> 			`	"I'm sorry, Anoo, but I really think Captain <first> is right. Fundamentally, I'm not the kind of person to be friendly with a group who starves and yet paints an exotic picture of itself. And in addition, I doubt there would be a way to support a family of two here; nuts by themselves are precious enough, let alone cookies. Hospital work can only pay so much.`

Reviewer: ```suggestion
			`	"I'm sorry, Anoo, but I really think Captain <first> is right. Fundamentally, I'm not the kind of person to be friendly with a group who starves and yet paints an exotic picture of itself. And in addition, I doubt there would be a way to support a family of two here; nuts by themselves are precious enough, let alone cookies. Hospital work here can only pay so much.`
```

				accept

			label reallyjoin
			action
				set "acorn delights: eeeya joined"
			`	"No pun intended, of course," Anoo laughs. Eeeya chuckles, and says, "Captain, I still need to dispose of my assets in normal Hai space before officially joining the Unfettered. Could you take me back to <planet> so that I can sell off the factory? And Anoo, I've got a surprise for you, still on the Captain's ship: 3 tons of acorn cookies!"`
			choice
				`	"Sure thing. I'd be glad to take you back."`
					accept
	on visit
		dialog "You forgot Eeeya or some of her mountains of cookies. Maybe they're in one of your escorts?"

mission "Acorn Delights: Part 5"
	name "Eeeya to Anoo"
	description "Bring Eeeya to <destination> to join the Unfettered and be near her granddaughter."
	landing
	clearance `The Darkcloak flight controller lets you land when they hear that Eeeya is joining the Unfettered.`
	source "Allhome"
	destination "Darkcloak"
	passengers 1
	to offer
		has "Acorn Delights: Part 4: done"
	on offer
		conversation
			branch uhai
				has "acorn delights: eeeya joined"
			action
				"salary: Acorn Delights" = 100

> 				log "Minor People" "Eeeya and Anoo" `Helped Eeeya, who runs a cookie shop called Acorn Delights, find her missing granddaughter Anoo. She ran off to join the Unfettered, but Eeeya wasn't interested in doing so. Received stocks for a one hundred credit salary for my efforts.`

Reviewer: ```suggestion
				log "Minor People" "Eeeya and Anoo" `Helped Eeeya, who runs a cookie shop called Acorn Delights, find her missing granddaughter Anoo.  After a long journey, we found her - she had gone to help the Unfettered as a trauma surgeon. Eeeya wasn't interested in joining the Unfettered, so brought her back home and received stocks for a one hundred credit salary for my efforts.`
```

			`	"We think of them as mindless brutes, but they're not! They have scientists, doctors, musicians, and writers. Our resources dwarf theirs, but they thrive with what little they have. Soon the Unfettered will have many more planets and almost unlimited resources. The era of scarcity will be over, and they'll have the power to choose their future.`
			`	"If enough good people join the Unfettered, we can change their minds and set them on a more productive path. They'll need good doctors."`
			`	"But-" Eeeya objects.`
			choice
				`	"Anoo is right; the Unfettered are prestigious. I would recommend joining them."`
					goto joinuhai
				`	"There's no good reason to join the Unfettered. What have they ever done except kill innocent Hai?"`
					goto "don't join"
			label joinuhai
			`	"Well... I really don't know... but how can I refuse when both my granddaughter and <first> tell me to?"`
			choice
				`	"Never mind, you're right. Joining the Unfettered is a bad idea after all."`

> 				`	"I stand by their guns. The Unfettered are honorable."`

Author: The point of changing "my" to "their" was to add a pun; something to break tension.

				log "Minor People" "Eeeya and Anoo" `Helped Eeeya, who runs a cookie shop called Acorn Delights, find her missing granddaughter Anoo. She ran off to join the Unfettered, but Eeeya wasn't interested in doing so. Received stocks for a two hundred credit salary for my efforts.`
			`Eeeya immediately gets off the ship and goes to check on the acorn cookie shop. A few minutes later, she is back with documents for you to sign.`
			`	"These will set you up with about a tenth of the stock in my company; you'll receive two hundred credits daily." As you sign the documents, she continues, "I'd just like to thank you again for helping me find Anoo. It means a lot." You check that the stocks have been transferred to your account, and Eeeya heads off.`
				decline
			label uhai
			`Eeeya heads off to sell Acorn Delights' shop while you finish the landing procedure. Soon, she is back. "Well, there was a Hai who paid thirty million credits for that small place. I wonder why he thought it was worth so much - I sure don't. But I won't argue with that nutty person." She seems much merrier now that Anoo has been found. "I'll be ready to go when you are."`
				accept
	on complete

> 		payment 1000000

Reviewer: This seems a bit high?

		log "Minor People" "Eeeya and Anoo" `Helped Eeeya, who runs a cookie shop called Acorn Delights, find her missing granddaughter Anoo. After a long journey, we found her - she had gone to help the Unfettered as a trauma surgeon. Brought Eeeya back to <origin> so she could sell the shop, but both Hai are together on <destination> now. Received a total of one million and three thousand credits for my efforts.`
		conversation
			`Eeeya leaves to go to the hospital, after paying you five hundred thousand credits and thanking you for the third time. But before you are allowed to leave your ship, an Unfettered security officer arrives. "Come with me<, unfettered title>, we have something to say to you."`

> 			`	"Why?" you ask. The Unfettered leads you to an office without answering. "We would like to thank you for recruiting a Hai - especially one as renowned as Chef Eeeya. You are being paid an additional five hundred thousand credits; do not think it is too much. You are dismissed." As you head back to <ship>, your communicator receives a receipt for the promised payment.`

Reviewer: Why would the Unfettered be so thrilled that a cookie baker is joining their ranks?

			choice
				`	"I'd be glad to take you back."`
				`	"I'll take you, but I'll notice that you haven't mentioned what we came so far for."`
			`	Eeeya smiles at her granddaughter. "Before I leave, I was just reminded about a present I have for you." She leads you and Anoo back to <ship>. "Captain, could you open the cargo hold and take us inside?" Anoo is astonished at the tons of acorn cookies waiting for her inside. "Thank you so much, grandma!"`
			choice
				`	"We should get going."`
			`	Eeeya turns to you and says, "You're right, we should. Thank you so much for reuniting us." She gives Anoo one last hug, and boards your ship.`
				accept

			label reallyjoin
			action
				set "acorn delights: eeeya joined"
			`	"No pun intended, of course," Anoo laughs. Eeeya chuckles, and says, "Captain, I still need to dispose of my assets in normal Hai space before officially joining the Unfettered. Could you take me back to <planet> so that I can sell off the factory? And Anoo, I've got a surprise for you, still on the Captain's ship: 3 tons of acorn cookies!"`
			choice

> 				`	"Sure thing. I'd be glad to take you back."`

Reviewer: You could add something here about the player being surprised Eeeya accepted joining the Unfettered so easily, which may provide an opportunity for her to explain why she's joining them. Perhaps have her mention here that she has few to no ties left in Hai space, and that's part of the reason she's deciding to join the Unfettered.

			`"Thank you, Captain, for making this journey. I apologize for not telling you more about me; the work I do is not something that I can reveal to any human who asks." He hands you <payment> and moves away. The surrounding Unfettered appear almost awed by his presence and consequentially give you a wide berth.`
		log "Minor People" "Hareek" `An old Hai who claims to foresee the future. Dropped him off in Unfettered space so he could spread his message there. He appears to be well-respected amongst the Unfettered.`

mission "Acorn Delights: Part 1"
	minor
	name "Acorns for Anoo"
	description "Bring Eeeya to <destination> to deliver <cargo> to her missing granddaughter."
	source "Allhome"
	destination "Stonebreak"
	passengers 1
	cargo "acorn cookies" 3
	to offer
		random < 35
		has "First Contact: Unfettered: offered"
	on offer
		conversation
			`As you pass by a kitchen supply store in the spaceport, a dishevelled elderly Hai looks towards you with a face of deep concern. She waves you down.`
			choice
				`	(See what she wants.)`
				`	(Ignore her.)`
					decline

> 			`	"My name is Eeeya, and I'm looking for my lost granddaughter, Anoo. She just passed the medical school, and I was going to give her three tons of cookies, but she's gone! Please help me find her."`

Reviewer: Perhaps mention here that she has some sort of accent, so her grammatical errors are less out of the blue and the reader doesn't think that they were mistakes.

Author: ```suggestion
			`	"My name is Eeeya, and I'm looking for my lost granddaughter, Anoo. She just passed the medical school, and I was going to give her three tons of cookies, but she's gone! Please help me find her." The Hai seems to have a strange accent, or perhaps she just doesn't speak English very well.`
```
I'm kind of on the fence about this one (why I haven't committed it yet) because it seems a bit awkward to be directly telling the player what language they're speaking. Thoughts?

Reviewer 2: Hai language is soft, fluid, and chittering. So what about emphasizing those characteristics as well as her uncertainty with articles somewhat like the below?

```suggestion
 
 			`	"My name is Eeeya, and I'm looking for my lost granddaughter, Anoo. She just passed the medical school, and I was going to give her three tons of cookies, but she's gone! Please help me find her." Her soft voice has a chittering edge much stronger than other Hai you've spoken with. She also hesitates before certain words, as if unsure whether to add a or the, giving her speech an unusual rhythm. `

```

> 			`	"My name is Eeeya, and I'm looking for my lost granddaughter, Anoo. She recently graduated from the medical school, and I was going to give her three tons of cookies, but she's gone! Please help me find her."`

Reviewer 2: ```suggestion
			`	"My name is Eeeya, and I'm looking for my lost granddaughter, Anoo. She just passed the medical school, and I was going to give her three tons of cookies, but she's gone! Please help me find her."`
```

> 			`	"My name is Eeeya, and I'm looking for my lost granddaughter, Anoo. She just passed the medical school, and I was going to give her three tons of cookies, but she's gone! Please help me find her." Her soft voice has a chittering edge much stronger than other Hai you've spoken with. She also hesitates before certain words, as if unsure whether to add a or the, giving her speech an unusual rhythm. `

Author: ```suggestion
			`	"My name is Eeeya, and I'm looking for my lost granddaughter, Anoo. She just passed the medical school, and I was going to give her three tons of cookies, but she's gone! Please help me find her." Her soft voice has a chittering edge much stronger than other Hai you've spoken with. She also hesitates before certain words, as if unsure whether to add a or the, giving her speech an unusual rhythm.`
```

				goto "actual questions"
			label questions
			`	"So please, take me to find my granddaughter."`
			label "actual questions"
			choice
				`	"Three tons of cookies!?"`
					goto tons
				`	"She's gone?"`
					goto gone
				`	"Why don't you ask the authorities for help?"`
					goto authorities
				`	"Can't your friends help you find her?"`
					goto friends
				`	"Let's go find her!"`
					goto accept
				`	"I have other places to be."`
					decline
			label tons

> 			`	"Well, I... I don't know... I cook when I'm worried, and Anoo has been missing for three months! I've vacuum-sealed all three tons of cookies. They should last a year or two. Maybe she'll share them with her friends? She has many friends! Tons of friends! They all miss her, but none are willing to help...`

Reviewer 2: ```suggestion
			`	"Well, I... I don't know... I cook when I'm worried, and Anoo has been missing for three months! I've well-sealed all three tons of cookies. They should last a year or two. Maybe she'll share them with her friends? She has many friends! Tons of friends! They all miss her, but none are willing to help...`
```

				goto questions
			label gone

> 			`	"Anoo graduated from a medical academy on Frostmark four months ago, near the top of her class! A prominent hospital on the Hai-home needed a new trauma surgeon and said she would be perfect for the job. She accepted the position and said she would be on vacation for a few weeks.`

Reviewer 2: ```suggestion
			`	"Anoo passed medical school on the Frostmark four months ago, near the top of her class! A famous hospital on the Hai-home needed a new trauma surgeon and said she would be perfect for the job. She accepted the position and said she would be on vacation for a few weeks.`
```

			`	"A month later, she told them she won't be coming, and nobody has heard from her since.`
				goto questions
			label authorities

> 			`	"I talked to the Central Medical Bureau, and they told me she got clearance for practicing medicine anywhere in Hai space or beyond. They've heard nothing from her since she got clearance. She has to register new prescriptions with the Bureau, so they should have heard from her.`

Reviewer 2: ```suggestion
			`	"I talked to the Central Medical Bureau, and they told me she got approval for practicing medicine anywhere in Hai space or beyond. They've heard nothing from her since she got approval. She has to register new medicines with the Bureau, so they should have heard from her.`
```

			`	"The police told me that several other people have reported Anoo missing, but they refused to tell me anything else. They said I had to talk to military, but they won't tell me anything.`
			`	"Maybe she's involved in something secret? Or illegal? No, she'd never do that. Where is she?`
				goto questions
			label friends
			`	"I have few friends, and none of them are interested in finding a missing Hai who could be anywhere. None of my customers want to help either, and her family is all dead.`
				goto questions
			label accept

> 			`	"She always talked about doing volunteer work for those who couldn't afford medical treatment. When she was little, her mother would take her to Stonebreak to help at local shelters. Let's start there."`

Reviewer 2: ```suggestion
			`	"She always talked about doing free work for those who couldn't afford medical treatment. When she was little, her mother would take her to Stonebreak to help at local shelters. Let's start there."`
```

				accept
	on visit
		dialog "You forgot Eeeya or some of her mountains of cookies. Maybe they're in one of your escorts?"

mission "Acorn Delights: Part 2"
	landing
	name "Acorns for Anoo"
	description "Bring Eeeya to <destination> to deliver <cargo> to her missing granddaughter."
	source "Stonebreak"
	destination "Cloudfire"
	passengers 1
	cargo "acorn cookies" 3
	to offer
		has "Acorn Delights: Part 1: done"
	on offer
		log "Took a Hai lady named Eeeya to Stonebreak to help find her granddaughter, who is named Anoo."
		conversation
			`You spend several hours flying Eeeya between the massive communities of Stonebreak. She checks every homeless shelter, volunteer ward, women's shelter, and low-income hospital. The story is always the same: Anoo hasn't been seen in years, though some remember the energetic little girl in her youth before her mother left.`

> 			`	After a few moments of crushing despair, Eeeya's ears perk up, and she says, "Since it's obvious she's not here, let's check on <destination> - she could be in military."`

Reviewer 2: ```suggestion
			`	After a few moments of crushing despair, Eeeya's ears perk up, and she says, "Since it's clear she's not here, let's check on <destination> - she could be in military."`
```

	source "Cloudfire"
	destination "Darkcloak"
	passengers 1
	cargo "acorn cookies" 3
	to offer
		has "Acorn Delights: Part 2: done"
	on offer
		conversation
			`Eeeya takes you to the visitor's office and asks to see the Cloudfire military commander. Upon refusal, Eeeya throws a fit; after an hour of her panic they agree to let her speak with the commander.`
			`	After a few minutes, she comes back looking despondent, and you lead her back to <ship>.`
			`	"They said she doesn't work for them. I think they know where she is, but they won't tell me anything.`
			`	"There's one more place we can try, but I don't think you're going to like it. She may have joined Unfettered Hai."`
			label questions
			choice
				`	"Why didn't you mention this earlier?"`
					goto mention
				`	"Why would a doctor join a band of warmongers?"`
					goto warmonger
				`	"As the honorable Captain <first> <last>, I say let's go find her!"`
					goto accept
				`	"I'd be happy to help."`
					goto accept
				`	"Unfettered space is too dangerous for me. You will have to find another captain."`
					decline
			label mention

> 			`	"I was afraid it would scare you away. Humans are terrified of our ships, and the Unfettered are deadly warriors. Besides, the other places were more likely. Her mother's family does have a tradition of joining the Unfettered, but I didn't think that would happen to her."`

Reviewer 2: ```suggestion
			`	"I was afraid it would scare you away. Humans are scared of our ships, and the Unfettered are deadly warriors. Besides, the other places were more likely. Many of her mother's family have joined the Unfettered in the past, but I didn't think that would happen to her."`
```

				goto questions
			label warmonger

> 			`	"Anoo was always sympathetic to the cause of the Unfettered Hai. Not their pointless war - she hated that - but she respected their drive to progress and their instinct for change. Her father's family tried to discourage her ideas, and we thought we succeeded, but... maybe...`

Reviewer 2: ```suggestion
			`	"Anoo was always sympathetic to the cause of the Unfettered Hai. Not their pointless war - she hated that - but she respected their drive to progress and their gift for change. Her father's family tried to stop her ideas, and we thought we succeeded, but... maybe...`
```

	source "Darkcloak"
	destination "Allhome"
	passengers 1
	to offer
		has "Acorn Delights: Part 3: done"
	on offer
		payment 3000
		log "Finally found Anoo. It turns out she joined the Unfettered. I am supposed to take Eeeya to <destination> now."
		conversation
			`You're ushered from the landing platform to a nearby hospital, where Eeeya finally manages to convey your intent. A few Unfettered Hai nearby are muttering in their language, and you keep hearing the Hai word for "cookies" as they watch Eeeya. Along the way, you notice an odd contrast of squalor and advanced machinery. Unlike their southern brethren, it seems the Unfettered value function and progress over aesthetics and comfort.`
			`	As you rise through the main tower of the hospital, you're surprised at how advanced it is. Despite their willingness to die in battle, the Unfettered seem to value the lives of the survivors. Perhaps this hospital is to heal them quickly so that they can fight again as soon as possible.`
			`	You reach a high floor filled with personal offices. Eeeya knocks on each door, apologizing when the Unfettered inside aren't who she's looking for, until you finally reach the very last room.`

> 			`	Eeeya opens this door, and runs to the Hai inside it - Anoo. A few minutes later, they begin to speak in Hai, then remember that you are in the room.`

Reviewer: What are they doing in those few minutes?
```suggestion
			`	Eeeya opens this door, and runs to the Hai inside it - Anoo. They begin to speak in Hai, then remember that you are in the room.`
```

			`	"Please, Anoo, let's speak in the human language. <first> came all this way for me, and it's only fair that Captain be able to understand what's going on."`
			`	The conversation then turns to Anoo's new occupation.`
			`	"Why did you come here, Anoo? The Unfettered hate us."`

> 			`	"They don't hate us in particular, grandma. They hate what we've become: frivolous, uninventive, and self-important. Our species is fading into history through a slow death of mediocrity. Most Unfettered see the warfare as our only salvation.`

Reviewer: Since Anoo's younger and well-educated, I think she should have a good grasp of the human language.
```suggestion
			`	"They don't hate us in particular, grandma. They hate what we've become: frivolous, uninventive, and self-important. Our species is fading into history through a slow death of mediocrity. Most Unfettered see warfare as our only salvation.`
```

> 			`	"We think of them as the mindless brutes, but they're not! They have scientists, doctors, musicians, and writers. Our resources dwarf theirs, but they thrive with what little they have. Soon the Unfettered will have many more planets and almost unlimited resources. The era of scarcity will be over, and they'll have the power to choose their future.`

Reviewer: ```suggestion
			`	"We think of them as mindless brutes, but they're not! They have scientists, doctors, musicians, and writers. Our resources dwarf theirs, but they thrive with what little they have. Soon the Unfettered will have many more planets and almost unlimited resources. The era of scarcity will be over, and they'll have the power to choose their future.`
```

			`	"If enough good people join the Unfettered, we can change their minds and set them on a more productive path. They'll need good doctors."`
			`	"But-" Eeeya objects.`
			choice
				`	"Anoo is right; the Unfettered are a noble people. I would recommend joining them."`
				`	"There's no good reason to join the Unfettered. What have they ever done except kill innocent Hai?"`
					goto "don't join"
			label "join unfettered"
			`	"Well... I really don't know..."`
			choice
				`	"Never mind, you're right. Joining the Unfettered is a bad idea after all."`
				`	"I stand by their guns. The Unfettered are honorable."`
					goto "really join"
			label "don't join"

> 			`	"I'm sorry, Anoo, but I really think the Captain <first> is right. I'm not the kind of person to be friendly with a group who starves and yet paints a colorful picture of itself. And in addition, I doubt there would be a way to support a family of two here; nuts by themselves are costly enough, let alone cookies. The hospital work here can only pay so much.`

Reviewer: I think 'romantic' might be more appropriate here?
```suggestion
			`	"I'm sorry, Anoo, but I really think the Captain <first> is right. I'm not the kind of person to be friendly with a group who starves and yet paints such a romantic picture of itself. And in addition, I doubt there would be a way to support a family of two here; nuts by themselves are costly enough, let alone cookies. The hospital work here can only pay so much.`
```

> 			`	"I'm sorry, Anoo, but I really think the Captain <first> is right. Fundamentally, I'm not the kind of person to be friendly with a group who starves and yet paints an exotic picture of itself. And in addition, I doubt there would be a way to support a family of two here; nuts by themselves are precious enough, let alone cookies. The hospital work here can only pay so much.`

Reviewer 2: ```suggestion
			`	"I'm sorry, Anoo, but I really think the Captain <first> is right. I'm not the kind of person to be friendly with a group who starves and yet paints an colorful picture of itself. And in addition, I doubt there would be a way to support a family of two here; nuts by themselves are costly enough, let alone cookies. The hospital work here can only pay so much.`
```

Author: ```suggestion
			`	"I'm sorry, Anoo, but I really think the Captain <first> is right. I'm not the kind of person to be friendly with a group who starves and yet paints a colorful picture of itself. And in addition, I doubt there would be a way to support a family of two here; nuts by themselves are costly enough, let alone cookies. The hospital work here can only pay so much.`
```

			`	"Captain, could you take me back to the <planet> please? I'd like to sign the forms and give you some stock in my Acorn Delights factory. In the meantime, here is <payment> for what you've done so far."`
			choice
				`	"I'd be glad to take you back."`
				`	"I'll take you, but I'll notice that you haven't mentioned what we came so far for."`
			`	Eeeya smiles at her granddaughter. "Before I leave, I was just reminded about a present I have for you." She leads you and Anoo back to <ship>. "Captain, could you open the cargo hold and take us inside?" Anoo is astonished at the tons of acorn cookies waiting for her inside. "Thank you so much, grandma!"`
			choice
				`	"We should get going."`
			`	Eeeya turns to you and says, "You're right, we should. Thank you so much for reuniting us." She gives Anoo one last hug, and boards your ship.`
				accept

			label "really join"
			action
				set "acorn delights: eeeya joined"

> 			`	"No pun intended, of course," Anoo laughs. Eeeya chuckles, and says, "Captain, I still need to dispose of my assets in the normal Hai space before officially joining Unfettered."`

Reviewer 2: ```suggestion
			`	"No pun intended, of course," Anoo laughs. Eeeya chuckles, and says, "Captain, I still need to sell my factory in the normal Hai space before officially joining Unfettered."`
```

				accept
	on visit
		dialog "You forgot Eeeya or some of her mountains of cookies. Maybe they're in one of your escorts?"

mission "Acorn Delights: Part 5"
	landing
	name "Eeeya to Anoo"
	description "Bring Eeeya to <destination> to join the Unfettered and be near her granddaughter."
	clearance `The Darkcloak flight controller lets you land when they hear that Eeeya is joining the Unfettered.`
	source "Allhome"
	destination "Darkcloak"
	passengers 1
	to offer
		has "Acorn Delights: Part 4: done"
	on offer
		conversation
			branch uhai
				has "acorn delights: eeeya joined"
			action
				"salary: Acorn Delights" = 100
				log "Minor People" "Eeeya and Anoo" `Helped Eeeya, who runs a cookie shop called Acorn Delights, find her missing granddaughter Anoo. After a long journey, we found her - she had gone to help the Unfettered as a trauma surgeon. Eeeya wasn't interested in joining the Unfettered, so brought her back home and received stocks for a one hundred credit salary for my efforts.`
			`Eeeya immediately gets off the ship and goes to check on the acorn cookie shop. A few minutes later, she is back with documents for you to sign.`

> 			`	"These will set you up with about a twentieth of the stock in my company; you'll receive the one hundred credits daily." As you sign the documents, she continues, "I'd just like to thank you again for helping me find Anoo. It means a lot to me." You check that the stocks have been transferred to your account, and Eeeya heads off.`

Reviewer: ```suggestion
			`	"These will set you up with about a twentieth of the stock in my company; you'll receive the one hundred credits daily." As you sign the documents, she continues. "I'd just like to thank you again for helping me find Anoo. It means a lot to me." You check that the stocks have been transferred to your account, and Eeeya heads off.`
```

				decline
			label uhai

> 			`Eeeya heads off to sell Acorn Delights' shop while you finish the landing procedure. Soon, she is back. "Well, there was a Hai who paid thirty million credits for that small place. I wonder why he thought it was worth so much - I sure don't. But I won't argue with that nutty person." She seems much merrier now that Anoo has been found. "I'll be ready to go when you are."`

Reviewer: How does she leave your ship before you've landed it?

Author: ```suggestion
			`Eeeya heads off to sell Acorn Delights' shop while you finish signing the documents to be officially landed. Soon, she is back. "Well, there was a Hai who paid thirty million credits for that small place. I wonder why he thought it was worth so much - I sure don't. But I won't argue with that nutty person." She seems much merrier now that Anoo has been found. "I'll be ready to go when you are."`
```
I meant something like this, the way you don't enter a country even when you've landed at an airport unless you go through security.
Would there be a better way to imply this than my change?

Reviewer: How about this?
```suggestion
			`Eeeya heads off to sell Acorn Delights' shop while you check in with spaceport security. Soon, she is back. "Well, there was a Hai who paid thirty million credits for that small place. I wonder why he thought it was worth so much - I sure don't. But I won't argue with that nutty person." She seems much merrier now that Anoo has been found. "I'll be ready to go when you are."`
```

			`"Thank you, Captain, for making this journey. I apologize for not telling you more about me; the work I do is not something that I can reveal to any human who asks." He hands you <payment> and moves away. The surrounding Unfettered appear almost awed by his presence and consequentially give you a wide berth.`
		log "Minor People" "Hareek" `An old Hai who claims to foresee the future. Dropped him off in Unfettered space so he could spread his message there. He appears to be well-respected amongst the Unfettered.`

mission "Acorn Delights: Part 1"
	minor
	name "Acorns for Anoo"
	description "Bring Eeeya to <destination> to deliver <cargo> to her missing granddaughter."
	source "Allhome"
	destination "Stonebreak"
	passengers 1
	cargo "acorn cookies" 3
	to offer
		random < 35
		has "First Contact: Unfettered: offered"
	on offer
		conversation
			`As you pass by a kitchen supply store in the spaceport, a dishevelled elderly Hai looks towards you with a face of deep concern. She waves you down.`
			choice
				`	(See what she wants.)`
				`	(Ignore her.)`
					decline

> 			`	"My name is Eeeya, and I'm looking for my lost granddaughter, Anoo. She recently graduated from medical school, and I was going to give her three tons of cookies, but she's gone! Please help me find her."`

Author: Take a look; I think I've updated this.

				`	"Why don't you ask the authorities to help?"`
					goto authorities
				`	"Let's go find her!"`
					goto accept
				`	"I have other places to be."`
					decline
			label tons
			`	"Well I... I don't know... I cook when I'm worried, and Anoo has been missing for three months! I've vacuum-sealed all three tons of cookies. They should last a year or two. Maybe she'll share them with her friends? She has many friends! Tons of friends! They all miss her...`
				goto questions
			label gone
			`	"Anoo graduated from a medical academy on Frostmark four months ago, near the top of her class! A prominent hospital on Hai-Home needed a new trauma surgeon and said she would be perfect for the job. She accepted the position and said she would be on vacation for a few weeks.`
			`	"A month later, she told them she won't be coming, and nobody has heard from her since.`
				goto questions
			label authorities
			`	"I talked to the Central Medical Bureau, and they told me she got clearance for practicing medicine anywhere in Hai space or beyond. They've heard nothing from her since she got clearance. She has to register new prescriptions with the Bureau, so they should have heard from her.`

> 			`	"The police told me that several other people have reported Anoo missing, but they refused tell me anything else. They said I had to talk to the military, and the military won't tell me anything.`

Reviewer 3: ```suggestion
			`	"The police told me that several other people have reported Anoo missing, but they refused to tell me anything else. They said I had to talk to the military, but the military won't tell me anything.`
```

			`	"Maybe she's involved in something secret? Or illegal? No, she'd never do that. Where is she?`
				goto questions
			label accept

> 			`	"She always talked about doing volunteer work for those who couldn't afford medical treatment. When she was little, her mother would take her to Stonebreak to help at local shelters. Let's start there."`

Reviewer 3: Honestly reading this I didnt realise she was coming until the end, mb it could be clearer? Or mb it's just me

					goto accept
				`	"Unfettered space is too dangerous for me. You will have to find another captain."`
					decline
			label mention
			`	"I was afraid it would scare you away. Humans are terrified of our ships, and the Unfettered are deadly warriors. Besides, the other places were more likely."`
				goto questions
			label warmonger
			`	"Anoo was always sympathetic to the cause of the Unfettered Hai. Not their pointless war - she hated that - but she respected their drive to progress and their instinct for change. Our family tried to discourage her ideas, and we thought we succeeded, but... maybe...`
			`	"A trauma surgeon for the Unfettered Hai? I wonder if she planned this years in advance?"`
				goto questions
			label accept
			`	"If Anoo went anywhere, it would be Darkcloak. The largest Unfettered hospital is there. Many soldiers come back from raids injured on Darkcloak. A trauma surgeon would have a lot of work to do."`
				accept
	on visit
		dialog "You forgot Eeeya or some of her mountains of cookies. Maybe they're in one of your escorts?"
			
mission "Acorn Delights: Part 5"
	landing
	name "Acorns for Anoo"

> 	description "Bring Eeeya to <destination> to deliver <cargo> to her missing granddaughter."

Reviewer 4: is cargo supposed to be defined under this mission tag?

Author: No. I forgot to change the description.

			`	"I'm sorry, Anoo, but I really think Captain <first> is right. While the Unfettered may be kind enough to shelter Hai who come bereaved from their own shots, they do have much starvation. And also, I can't sell cookies here. I will come visit occasionally though.`
			`	"Captain, could you take me back to <planet> please? I'd like to sign the forms and give you some stock in my Acorn Delights factory. In the meantime, here is <payment> for what you've done so far."`
			choice
				`	"I'd be glad to take you back."`
				`	"I'll take you, but I'll notice that you haven't mentioned what we came so far for."`
			`	Eeeya smiles at her granddaughter. "Before I leave, I was just reminded about a present I have for you." She leads you and Anoo back to <ship>. "Captain, could you open the cargo hold and take us inside?" Anoo is astonished at the tons of acorn cookies waiting for her inside. "Thank you so much, grandma!"`
			choice
				`	"We should get going."`
			`	Eeeya turns to you and says, "You're right, we should. Thank you so much for reuniting us." She gives Anoo one last hug, and boards your ship.`
				accept

			label reallyjoin
			action
				set "acorn delights: eeeya joined"

> 			`	"No pun intended, of course," Anoo laughs. Eeeya chuckles, and says, "Captain, I still need to dispose of my assets in normal Hai space before officially joining the True Hai, which I have now decided to do. Could you take me back to <planet> so that I can sell off the factory? And Anoo, I've got a surprise for you, still on the Captain's ship: 3 tons of acorn cookies!"`

Reviewer 4: Should this use the <cargo> tag in the final sentence?

Author: No, because it's in the `on offer` section of a mission that doesn't have cargo.

			`	"These will set you up with about a tenth of the stock in my company; you'll receive two hundred and fifty credits daily." As you sign the documents, she continues, "I'd just like to thank you again for helping me find Anoo. It means a lot." You check that the stocks have been transferred to your account, and Eeeya heads off.`
				decline
			label uhai
			`Eeeya heads off to sell Acorn Delights' shop while you finish the landing procedure. Soon, she is back. "Well, there was a Hai who paid thirty million credits for that small place. I wonder why he thought it was worth so much - I sure don't. But I won't argue with that nutty person." She seems much merrier now that Anoo has been found. "I'll be ready to go when you are."`
				accept
	on complete
		payment 1000000
		conversation
			`Eeeya leaves to go to the hospital, after paying you five hundred thousand credits and thanking you for the third time. But before you are allowed to leave your ship, an Unfettered security officer arrives. "Come with me<, unfettered title>, we have something to say to you."`

> 			`	"Why?" you ask. The Unfettered leads you to an office without answering. "We would like to thank you for recruiting a Hai - especially one as renowned as Chef Eeeya. You are being paid an additional five hundred thousand credits; do not think it is too much. You are dismissed." As you head back to <ship>, your communicator receives a receipt for the promised payment.`

Reviewer 4: The words say "five hundred thousand" but I don't see any 500000 credits anywhere in the numbers.

Author: You get paid a million; 500000 by the Unfettered officer and 500000 by Eeeya.

			`"Thank you, Captain, for making this journey. I apologize for not telling you more about me; the work I do is not something that I can reveal to any human who asks." He hands you <payment> and moves away. The surrounding Unfettered appear almost awed by his presence and consequentially give you a wide berth.`
		log "Minor People" "Hareek" `An old Hai who claims to foresee the future. Dropped him off in Unfettered space so he could spread his message there. He appears to be well-respected amongst the Unfettered.`

mission "Acorn Delights: Part 1"
	minor
	name "Acorns for Anoo"
	description "Bring Eeeya to <destination> to deliver <cargo> to her missing granddaughter."
	source "Allhome"
	destination "Stonebreak"
	passengers 1
	cargo "acorn cookies" 3
	to offer
		random < 85
		has "First Contact: Unfettered: offered"
	on offer
		conversation
			`As you pass by a kitchen supply store in the spaceport, a dishevelled elderly Hai looks towards you with a face of deep concern. She waves you down.`
			choice
				`	(See what she wants.)`
				`	(Ignore her.)`
					decline

> 			`	"My name is Eeeya, and I'm looking for my lost granddaughter, Anoo. She recently graduated from medical school, and I was going to give her three tons of cookies, but she's gone! Please help me find my granddaughter."`

Reviewer: It looks like a typo - what does three 'e's convey that two don't?

> 			`	"My name is Eeeya, and I'm looking for my lost granddaughter, Anoo. She recently graduated from medical school, and I was going to give her three tons of cookies, but she's gone! Please help me find my granddaughter."`

Author: That it's a Hai name, not a human one (Eeya is a Filipino name)

			`	"Maybe she's involved in something secret? Or illegal? No, she'd never do that. Where is she?`
				goto questions
			label accept
			`	"She always talked about doing volunteer work for those who couldn't afford medical treatment. When she was little, her mother would take her to Stonebreak to help at local shelters. Let's start there."`
				accept
	on visit
		dialog "You forgot Eeeya or some of her mountains of cookies. Maybe they're in one of your escorts?"

mission "Acorn Delights: Part 2"
	landing
	name "Acorns for Anoo"
	description "Bring Eeeya to <destination> to deliver <cargo> to her missing granddaughter."
	source "Stonebreak"
	destination "Alta Hai"
	passengers 1
	cargo "acorn cookies" 3
	to offer
		has "Acorn Delights: Part 1: done"
	on offer
		log "Took a Hai lady named Eeeya to Stonebreak to help find her granddaughter, who is named Anoo."
		conversation
			`You spend several hours flying Eeeya between the massive communities of Stonebreak. She checks every homeless shelter, volunteer ward, woman's shelter, and low-income hospital. The story is always the same: Anoo hasn't been seen in years, though some remember the energetic little girl in her youth.`

> 			`	After a few moments of crushing despair, Eeeya's ears perk up, and she says, "Sometimes very gifted students are allowed to study with the Quarg for a year after they graduate. There are trillions of Quarg on Alta Hai, so it wouldn't be surprising if she got lost. Let's go there next."`

Author: I think "very gifted" might imply that.
How about this though?
```suggestion
			`	After a few moments of crushing despair, Eeeya's ears perk up, and she says, "Sometimes extremely gifted students are allowed to study with the Quarg for a year after they graduate, though still only about Hai technology. There are trillions of Quarg on Alta Hai, so it wouldn't be surprising if she got lost. Let's go there next."`
```

> 			`	After a few moments of crushing despair, Eeeya's ears perk up, and she says, "Sometimes very gifted students are allowed to study with the Quarg for a year after they graduate. There are trillions of Quarg on Alta Hai, so it wouldn't be surprising if she got lost. Let's go there next."`

Reviewer: Why would the Quarg teach the Hai about their own technology?
To be honest I'd just remove this part of the mission.

	landing
	clearance `After hearing the name Eeeya, the Darkcloak flight controller excitedly responds, "Acorn Delights!" and you're granted landing clearance.`
	source "Cloudfire"
	destination "Darkcloak"
	passengers 1
	cargo "acorn cookies" 3
	to offer
		has "Acorn Delights: Part 3: done"
	on offer
		log "Eeeya's granddaughter Anoo still proves difficult to find. Checked for her on Alta Hai and Cloudfire."
		conversation
			`Eeeya takes you to the visitor's office and asks to see the Cloudfire military commander. Upon refusal, Eeeya throws a fit; after an hour of her panic they agree to let her speak with the commander.`
			`	After a few minutes, she comes back looking despondent, and you lead her back to the <ship>.`
			`	"They said she doesn't work for them. I think they know where she is, but they won't tell me anything.`
			`	"There's one more place we can try, but I don't think you're going to like it. She may have joined the Unfettered Hai."`
			label questions
			choice
				`	"Why didn't you mention this earlier?"`
					goto mention
				`	"Why would a doctor join a band of warmongers?"`
					goto warmonger

> 				`	"As the honorable captain <first> <last>, I say let's go find her!"`

Author: ```suggestion
				`	"As the honorable Captain <first> <last>, I say let's go find her!"`
					accept
				`	"I'd be happy to help."`
```

	source "Darkcloak"
	destination "Allhome"
	passengers 1
	to offer
		has "Acorn Delights: Part 4: done"
	on offer
		payment 3000
		log "Finally found Anoo. It turns out she joined the Unfettered. I am supposed to take Eeeya to <destination> now."
		conversation
			`You're ushered from the landing platform to a nearby hospital when Eeeya finally manages to convey your intent. A few Unfettered Hai nearby are muttering in their language, and you keep hearing the Hai word for "cookies" as they watch Eeeya. Along the way, you notice an odd contrast of squalor and advanced machinery. Unlike their southern brethren, it seems the Unfettered value function and progress over aesthetics and comfort like their southern brethren.`
			`	As you rise through the main tower of the hospital, you're surprised at how advanced it is. Despite their willingness to die in battle, the Unfettered seem to value the lives of the survivors. Perhaps this hospital is to heal them quickly to fight again.`
			`	You reach a high floor filled with personal offices. Eeeya knocks on each door, apologizing when the Unfettered inside aren't who she's looking for, until you finally reach the very last room.`

> 			`	Eeeya opens this door, and runs to the Hai inside it - Anoo. A few minutes later, they begin to speak. Eventually the conversation turns to Anoo's new occupation.`

Author: ```suggestion
			`	Eeeya opens this door, and runs to the Hai inside it - Anoo. A few minutes later, they begin to speak, surprisingly in a language you understand. Eventually the conversation turns to Anoo's new occupation.`
```

			`	"Why did you come here, Anoo? The Unfettered hate us."`
			`	"They don't hate us in particular, grandma. They hate what we've become: frivolous, uninventive, and self-important. Our race is fading into history through a slow death of mediocrity. Most Unfettered see warfare as our only salvation.` # "us in particular" should be changed to "us" and italicized if/when support for that is ever implemented.
			`	"We think of them as mindless brutes, but they're not! They have scientists, doctors, musicians, and writers. Our resources dwarf theirs, but they thrive with what little they have. Soon the Unfettered will have many more planets and almost unlimited resources. The era of scarcity will be over, and they'll have the power to choose their future.`
			`	"If enough good people join the Unfettered, we can change their minds and set them on a more productive path. They'll need good doctors."`
			`	"But-" Eeeya objects.`
			choice

> 				`	"Anoo is right; the Unfettered are prestigious. I would recommend joining them."`

Author: Same here, but this is the option you choose if you want her to become one of them. If the player is wholly on the side of the Unfettered, doesn't it make sense for them to use grand words to describe them?

> 				`	"Anoo is right; the Unfettered are prestigious. I would recommend joining them."`

Reviewer: I don't think even the Unfettered themselves would use the word prestigious to describe themselves. Prestigious implies status, but part of the Unfettered's deal is that they want to rebel against the traditional markers of status in Hai society - stability, comfort, peacefulness, not rocking the boat. They're the rejects and castoffs that couldn't fit into Hai culture, and they hold open disdain for the Hai Elders and other leaders of Hai society for falling into stagnation.

> 				`	"Anoo is right; the Unfettered are prestigious. I would recommend joining them."`

Author: What word would you use, than?

> 				`	"Anoo is right; the Unfettered are prestigious. I would recommend joining them."`

Author: Perhaps "noble"?

> 				`	"Anoo is right; the Unfettered are prestigious. I would recommend joining them."`

Reviewer: Yeah, that sounds better.

					goto joinuhai
				`	"There's no good reason to join the Unfettered. What have they ever done except kill innocent Hai?"`
					goto "don't join"
			label joinuhai
			`	"Well... I really don't know... but how can I refuse when both my granddaughter and <first> tell me to?"`
			choice
				`	"Never mind, you're right. Joining the Unfettered is a bad idea after all."`

> 				`	"I stand by my guns. The Unfettered are honorable."`

Author: ```suggestion
				`	"I stand by their guns. The Unfettered are honorable."`
```
```suggestion
				`	"I stand by my guns. The Unfettered are honorable."`
```

					goto reallyjoin
			label "don't join"

> 			`	"I'm sorry, Anoo, but I really think Captain <first> is right. While the Unfettered may be kind enough to shelter Hai who come bereaved from their own shots, they do have much starvation. And also, I can't sell cookies here. I will come visit occasionally though.`

Author: ```suggestion
			`	"I'm sorry, Anoo, but I really think Captain <first> is right. Fundamentally, I'm not the kind of person to be friendly with a group who starves and yet paints an exotic picture of itself. And in addition, I doubt there would be a way to support a family of two here; nuts by themselves are precious enough, let alone cookies. Hospital work can only pay so much.`
```

> 			`	"Captain, could you take me back to <planet> please? I'd like to sign the forms and give you some stock in my Acorn Delights factory. In the meantime, here is <payment> for what you've done so far."`

Reviewer: Why is she giving me stocks in my company? She barely knows me.

Author: You've helped her find something precious, and interacted several times.
I think it is possible to say she knows you, especially given the fact that she has just taken your advice on a life-changing decision.

Also, she has no way to pay you besides that; and what you've done is supremely important to her.

			choice
				`	"I'd be glad to take you back."`
				`	"I'll take you, but I'll notice that you haven't mentioned what we came so far for."`
			`	Eeeya smiles at her granddaughter. "Before I leave, I was just reminded about a present I have for you." She leads you and Anoo back to <ship>. "Captain, could you open the cargo hold and take us inside?" Anoo is astonished at the tons of acorn cookies waiting for her inside. "Thank you so much, grandma!"`
			choice
				`	"We should get going."`
			`	Eeeya turns to you and says, "You're right, we should. Thank you so much for reuniting us." She gives Anoo one last hug, and boards your ship.`
				accept

			label reallyjoin
			action
				set "acorn delights: eeeya joined"

> 			`	"No pun intended, of course," Anoo laughs. Eeeya chuckles, and says, "Captain, I still need to dispose of my assets in normal Hai space before officially joining the True Hai, which I have now decided to do. Could you take me back to <planet> so that I can sell off the factory? And Anoo, I've got a surprise for you, still on the Captain's ship: 3 tons of acorn cookies!"`

Reviewer: What pun?

> 			`	"No pun intended, of course," Anoo laughs. Eeeya chuckles, and says, "Captain, I still need to dispose of my assets in normal Hai space before officially joining the True Hai, which I have now decided to do. Could you take me back to <planet> so that I can sell off the factory? And Anoo, I've got a surprise for you, still on the Captain's ship: 3 tons of acorn cookies!"`

Reviewer: How are you able to convince someone to make this life-altering decision in a few minutes when you barely even know them?

Author: It's not just you; the most important person in Eeeya's life is her niece (and first last is likely the second). The player is just enough to tip the balance.

Reviewer: Why is the player the second most important person in her life? I know that first last has been helpful to her, but doesn't she have any other family or friends in Hai space? How is some random human captain able to convince an old cookie baker to up and leave her entire life behind for Unfettered space?

Author: I've tried to remedy this somewhat in eef72e37fd38f9eab567a1baeca488f5bebd0a0c. It may feel a bit contrived; let me know what you think.

Author: Essentially, the captain is one of the most important people because everyone else has either abandoned her in the hour of need or died.

			`Eeeya immediately gets off the ship and goes to check on the acorn cookie shop. A few minutes later, she is back with documents for you to sign.`
			`	"These will set you up with about a tenth of the stock in my company; you'll receive two hundred credits daily." As you sign the documents, she continues, "I'd just like to thank you again for helping me find Anoo. It means a lot." You check that the stocks have been transferred to your account, and Eeeya heads off.`
				decline
			label uhai
			`Eeeya heads off to sell Acorn Delights' shop while you finish the landing procedure. Soon, she is back. "Well, there was a Hai who paid thirty million credits for that small place. I wonder why he thought it was worth so much - I sure don't. But I won't argue with that nutty person." She seems much merrier now that Anoo has been found. "I'll be ready to go when you are."`
				accept
	on complete
		payment 1000000

> 		log "Minor People" "Eeeya and Anoo" `Helped Eeeya, who runs a cookie shop called Acorn Delights, find her missing granddaughter Anoo. After a long journey, we found her - she had gone to help the Unfettered as a trauma sergeon. Brought Eeeya back to <origin> so she could sell the shop, but both Hai are together on <destination> now. Received a total of one million and three thousand credits for my efforts.`

Reviewer: ```suggestion
		log "Minor People" "Eeeya and Anoo" `Helped Eeeya, who runs a cookie shop called Acorn Delights, find her missing granddaughter Anoo. After a long journey, we found her - she had gone to help the Unfettered as a trauma surgeon. Brought Eeeya back to <origin> so she could sell the shop, but both Hai are together on <destination> now. Received a total of one million and three thousand credits for my efforts.`
```

			label reallyjoin
			action
				set "acorn delights: eeeya joined"
			`	"No pun intended, of course," Anoo laughs. Eeeya chuckles, and says, "Captain, I still need to dispose of my assets in normal Hai space before officially joining the Unfettered."`
			choice

> 				`	"Why did you change your mind so quickly?"`

Reviewer: I was thinking that this choice option could be combined with the choice at line 2277, in order to avoid single option choices.
